commands/export.go

```diff
@@ -351,15 +352,18 @@ func transactionWithState(hash string, network *config.ExportNetwork) (types.Tra
 	return tx, state, networkId, nil
 }
 
-func contractsWithConfig(networkId string, objects []*model.StateObject) ([]truffle.Contract, *payloads.Config, error) {
+func contractsWithConfig(
+	networkId string,
+	objects []*model.StateObject,
+) ([]providers.Contract, *payloads.Config, error) {
 	logrus.Info("Collecting contracts...")
 
-	truffleConfig, err := MustGetTruffleConfig()
+	truffleConfig, err := deploymentProvider.MustGetConfig()
```

Comment:
Probably rename this to something not called `truffleConfig` as it is a generic config now.